stix_shifter_modules/carbonblack/stix_transmission/api_client.py

```diff
@@ -6,7 +6,8 @@ class APIClient():
     PROCESS_ENDPOINT = 'process'
 
     def __init__(self, connection, configuration):
-        self.endpoint_start = 'api/v1/'
+        self.endpoint_start_v1 = 'api/v1/'  # Uses API v1 for `ping` and `processes search` endpoints.
+        self.endpoint_start_v4 = 'api/v4/'  # Uses API v4 for `events search` endpoint.
```

Comment:
Sure, will do, since we don't have a specific connector for each version of CBR, we use the latest, we were recommended by MSS to use the v4 for events, means EDR >= 6.1

Reply:
@mdazam1942 I have added to readme: 
> Notice: If "Include Process Events?" option is enabled, CarbonBlack module searches events API endpoint. Enabling this option requires Carbon Black Response of version >= 6.1.

Not sure how to add this in the KC

Reply:
Thanks. I will take care of the KC.